index.html

```diff
@@ -2385,24 +2392,281 @@ <h2>
+          70776f6534737769726c733465626468            #       "pwoe4swirls4ebdh"
+          733269                                      #       "s2i"
+  </pre>
+
+</section>
+
+
+
+<section>
+  <h4>Consumption</h4>
+<p>  When consuming DID Documents that are represented as CBOR, in addition to the suggestions in section 3.9 of the CBOR [[RFC7049]] specification for deterministic mappings the following constraints of the DID Document model MUST be followed:  </p> 
+  <ul>
+  <li>The keys in every map must be sorted lowest value to highest. Sorting is performed on the bytes of the representation of the keys.</li>
+  <li>Indefinite-length items must be made into definite-length items.</li>
+</ul>
+</section>
+
+<section>
+  <h4>CBOR Extensibility</h4>
+  <p>In CBOR, one point of extensibility is with the use of CBOR tags. [[RFC7049]]  defines a basic set of data types, as
+    well as a tagging mechanism that enables extending the set of data types supported via an IANA registry [[CBOR-TAG-Registry]]. This allows for tags to enhance the semantic description of the data that follows. 
+  </p>
+
+  <section>
+      <h2>DagCBOR</h2>
+     <p>DagCBOR is a further restricted subset of CBOR for representing the DID Document as a Directed Acyclic Graph model using canonical CBOR encoding as noted above with additional constraits.  DagCBOR requires that there exist a single way of encoding any given object, and that encoded forms contain no superfluous data that may be ignored or lost in a round-trip decode/encode. When producing and consuming DID Documents representing in DagCBOR the following rules MUST be followed</p>
+     
+     <ul>
+       <li>Use no CBOR tags other than the CID tag (42)</li>
+     </ul>
+     
+     <pre class="example nohighlight" title="DID Document as DagCBOR same as the previous example, but serialized to JSON for easy readability">{ "@context": "https://www.w3.org/ns/did/v1",
+  "authentication": [
+    "did:example:12D3KooWMHdrzcwpjbdrZs5GGqERAvcgqX3b5dpuPtPa9ot69yew;key-id=bafyreicubtx5wqo3nosc4cazrkctfhwd6rewezgpwoe4swirls4ebdhs2i"
+  ],
+  "created": "2018-12-01T03:00:00Z",
+  "id": "did:example:12D3KooWMHdrzcwpjbdrZs5GGqERAvcgqX3b5dpuPtPa9ot69yew",
+  "proof": {
+    "created": "2020-05-01T03:00:02Z",
+    "creator": "did:example:12D3KooWMHdrzcwpjbdrZs5GGqERAvcgqX3b5dpuPtPa9ot69yew; example:key=id=bafyreicubtx5wqo3nosc4cazrkctfhwd6rewezgpwoe4swirls4ebdhs2i",
+    "signatureValue": "o9r6LxgoGN8FoaeeUA6EdDcv12GvDzFEmCgjWzvpur2YSQyA8W2r0SSWUK+nH5tMqzaFLun6wwZ1Eot37amGDg==",
+    "type": "ed25519Signature2018"
+  },
+  "publicKey": [
+    {
+      "curve": "ed25519",
+      "expires": "2019-12-01T03:00:00Z",
+      "id": "bafyreicubtx5wqo3nosc4cazrkctfhwd6rewezgpwoe4swirls4ebdhs2i;example:key=idbafyreicubtx5wqo3nosc4cazrkctfhwd6rewezgpwoe4swirls4ebdhs2i",
+      "publicKeyBase64": "qmz7tpLNKKKdl7cD7PbejDiBVp7ONpmZbfmc7cEK9mg=",
+      "type": "EdDsaPublicKey"
```

Comment:
not a registered public key type AFAIK

Reply:
I'll update the example and use proper CBOR tag.